src/common/utils/kaltura-params.js

```diff
@@ -93,6 +93,15 @@ function getReferrer(): string {
   return referrer;
 }
 
+/**
+ * @return {string} - The referrer after URIComponent encoded
+ * @private
+ */
+function getEncodedReferrer(): string {
```

Comment:
if it's not relevant to `kaltura-params.js` move it to `plugins-config.js`.

Reply:
fixed

Reply:
@yairans please check approve